_data/links.yml

```
@@ -23,6 +23,14 @@
   - 'background-size: 90%'
   - 'background-position-y: 25%'
   
+- name: EPITA Study Abroad
+  href: https://epitafr.sharepoint.com/sites/EPITAStudyAbroad
+  class: tile-medium text-dark
+  style:
+  - 'background-color: #fff'
+  - 'background-image: url(https://epitafr.sharepoint.com/sites/EPITAStudyAbroad/SiteAssets/__sitelogo__sans-titre.png)'
```

```suggestion
  - 'background-image: url(https://cdn.matiboux.com/images/epita.it/odyssee.jpg)'
```

Maybe consider using the same image as the Odyssée tile since Odyssée and EPITA Study Abroad both use the same logo.

---

Isn't that just the regular EPITA logo though? You can probably just use the same logo as the EPITA tile in that case?

---

Commit a8816e0 solves this.